src/order/conditionally_complete_lattice.lean

```diff
@@ -463,9 +483,17 @@ end conditionally_complete_linear_order
 
 section conditionally_complete_linear_order_bot
 
-lemma cSup_empty [conditionally_complete_linear_order_bot α] : (Sup ∅ : α) = ⊥ :=
+variables [conditionally_complete_linear_order_bot α]
+
+lemma cSup_empty : (Sup ∅ : α) = ⊥ :=
```

Comment:
isn't this weaker than the original `with_bot.cSup_empty`? or does `with_bot` upgrade a lattice all the way to a `linear_order_bot`?